csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/FailFastMetaDataValidator.scala

```diff
@@ -60,16 +60,10 @@ trait FailFastMetaDataValidator extends MetaDataValidator {
     validateRows()
   }
 
-  private def validateRow(row: Row, schema: Schema): MetaDataValidation[Any] = {
```

Comment:
What are these checks removed? I can see that you are trying to move them into the base class, but the checks between FailFastMetaDataValiator and AllErrorsMetadataValidator are not the same, which is why they were not in the base class.